bazel/BUILD

```diff
@@ -199,6 +199,11 @@ config_setting(
     values = {"define": "path_normalization_by_default=true"},
 )
 
+config_setting(
+    name = "enable_legacy_codecs_in_test",
```

Comment:
in_integration_tests here and for the #define name?

Reply:
done